lib/portage/package/ebuild/doebuild.py

```diff
@@ -2856,6 +2850,48 @@ def _reapply_bsdflags_to_image(mysettings):
         )
 
 
+def _inject_libc_dep(build_info_dir, mysettings):
+    #
+    # We could skip this for non-binpkgs but there doesn't seem to be much
+    # value in that, as users shouldn't downgrade libc anyway.
+    injected_libc_depstring = []
+    for libc_realized_atom in find_libc_deps(
+        QueryCommand.get_db()[mysettings["EROOT"]]["vartree"].dbapi, True
+    ):
+        if pkgcmp(mysettings.mycpv, libc_realized_atom) is not None:
+            # We don't want to inject deps on ourselves (libc)
+            injected_libc_depstring = []
+            break
+
+        injected_libc_depstring.append(f">={libc_realized_atom}")
+
+    rdepend_file = os.path.join(build_info_dir, "RDEPEND")
+    # Slurp the existing contents because we need to mangle it a bit
+    # It'll look something like (if it exists):
+    # ```
+    # app-misc/foo dev-libs/bar
+    # <newline>
+    # ````
+    rdepend = []
```

Comment:
Can initialize this to an empty string, None, or False since it's always reassigned later.